### PR TITLE
Test/serverApi

### PR DESCRIPTION
**Pull request by anishpawaskar**
**Add test for custom hooks created by RTK query in serverApi**

## Written test for the following RTK-custom Hooks:
- useArchiveMemberMutation
- useGetContributionsQuery
- useGetMembersQuery
- useGetUserActiveTaskQuery
- useGetUserQuery
- useGetSelfDetailsQuery
- useUpdateMemberRoleMutation
- useUpdateTaskStatusMutation

package json - members-site - Visual Studio Code 12-08-2023 11_39_23 (2)

### CODE REVIEW

File: src/test/functions/useGetMembers.test.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  return <Provider store={store}>{children}</Provider>;
+}
+
+describe("useGetMembers", () => {
+  test("it should returns members", async () => {
+    const { result } = renderHook(() => useGetMembers(), {
+      wrapper: Wrapper,
+    });
+
+    const { result: membesResult, waitForNextUpdate } = renderHook(
```
Comment (kotesh-arya) [inline src/test/functions/useGetMembers.test.tsx:30]: `membesResult` ??
Please fix this typo :)

**Review by kotesh-arya — COMMENTED** [review]
So, you're writing tests for multiple hooks which were already written. Can you update the PR-title to a bit more descriptive and the Description too, with whichever files you're writing tests for and Do let me know If this needs to be discussed 👍🏻

**Review by kotesh-arya — APPROVED** [review]

File: src/mocks/db/user.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+incompleteUserDetails: true,
+github_display_name: "Anish Pawaskar",
+github_id: "anishpawaskar",
+roles: {
+archived: false,
+in_discord: false,
+super_user: false
+},
+username: "anish"
+}
```
Comment (vinayak-trivedi) [inline src/mocks/db/user.ts:27]: can we fix the indentation here?

**Review by vinayak-trivedi — COMMENTED** [review]
overall looks good to me, just fix the indentation everywhere please

**Review by vinayak-trivedi — APPROVED** [review]
LGTM

**Pull request merged**